backend/src/org/commcare/xml/SuiteParser.java

```diff
@@ -33,6 +33,8 @@
     ResourceTable table;
     String resourceGuid;
 
+    private boolean skipResources = false;
+
     public SuiteParser(InputStream suiteStream, ResourceTable table, String resourceGuid) throws IOException {
```

Comment:
Will you add a class level doc? Something saying that this is used to used to build menus, entry actions, display details, get the app version, and install resources. I found myself wondering why we even run the suite parser on app init, and having such documentation would have answered it for me.